MobiFlight/MobiFlightCustomDevice.cs

```diff
@@ -3,6 +3,7 @@
 using System.Data.SqlTypes;
 using System.Drawing;
 using System.Linq;
+using System.Runtime.Remoting.Metadata.W3cXsd2001;
```

Comment:
What does this namespace provide? 🤔 I've never heard of it before.

Reply:
You are awesome! I removed all unused using... (99%)

Reply:
You should have seen how hard I was squinting at the code trying to figure out what it was for 😂